test/controllers/SafeNavigationBaseController.test.ts

```diff
@@ -161,7 +161,7 @@ describe('Safe navigation base controller', () => {
 
         it ('should not store navigation pass for already visited page that user is about to be redirected to', () => {
             const sessionStore = createSubstituteOf<SessionStore>(substitute => {
-                substitute.store(Arg.any()).returns(EitherUtils.wrapValue('OK'));
+                substitute.store(Arg.any());
```

Comment:
```suggestion
                substitute.store(Arg.any(), Arg.any(), Arg.any()).resolve('OK');
```

Reply:
TS2345: Argument of type '"OK"' is not assignable to parameter of type 'void'.